### PR TITLE
Add documentation for integration of Vallox ventilation units

### PR DESCRIPTION
**Pull request by andre-richter**
**Description:**

Accompanying documentation for the new integration PR for Vallox ventilation units.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24660
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/vallox.markdown [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.95
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/vallox.markdown]: Please remove the `redirect_from`, this is only for existing pages.

File: source/_components/vallox.markdown [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+description: "Instructions on how to integrate Vallox ventilation units into Home Assistant."
+date: 2019-06-19 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.95
```
Comment (klaasnicolaas) [inline source/_components/vallox.markdown]:
```suggestion
ha_release: 0.96
```

File: source/_components/vallox.markdown [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  required: true
+  type: string
+name:
+  description: The name of this device as you want to see it in Home Assistant.
+  required: false
+  default: Vallox
+  type: string
+{% endconfiguration %}
+
+## Fan Services
```
Comment (klaasnicolaas) [inline source/_components/vallox.markdown]:
```suggestion
## {% linkable_title Fan Services %}
```

File: source/_components/vallox.markdown [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_boost` %}
+
+Set the fan speed of the `Boost` profile.
+
+| Service data attribute | Optional | Description                                     |
+|------------------------|----------|-------------------------------------------------|
+| `entity_id`            |       no | Entitiy id of the fan, e.g.`fan.vallox`.        |
+| `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+## Sensors
```
Comment (klaasnicolaas) [inline source/_components/vallox.markdown]: Same

File: source/_components/vallox.markdown [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Also, there is a sensor platform that exposes a number of relevant metrics like fan speed, various air temperatures and humidity.
+
+### {% linkable_title Configuration %}
+
+```yaml
+vallox:
+  host: IP_ADDRESS
+```
+
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_components/vallox.markdown:35]: Missing the `sensor` config variable

File: source/_components/vallox.markdown [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.96
+ha_iot_class: Local Polling
+---
+
+The `vallox` component lets you control any Vallox ventilation unit that is supported by the [vallox_websocket_api](https://github.com/yozik04/vallox_websocket_api) (follow the link for a list of supported units).
+
+The component's fan platform allows you to either turn on/off the complete unit via toggle switch, or select a ventilation profile from a dropdown menu and therefore indirectly the fan speed. The four standard Vallox profiles are provided:
```
Comment (frenck) [inline source/_components/vallox.markdown]:
```suggestion
The component's fan platform allows you to either turn on/off the complete unit via the toggle switch or select a ventilation profile from a dropdown menu and therefore indirectly the fan speed. The four standard Vallox profiles are provided:
```

File: source/_components/vallox.markdown [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+- `At Home`
+- `Away`
+- `Boost`
+- `Fireplace`
+
+Also, there is a sensor platform that exposes a number of relevant metrics like fan speed, various air temperatures and humidity.
+
+### {% linkable_title Configuration %}
```
Comment (frenck) [inline source/_components/vallox.markdown:28]:
```suggestion
## {% linkable_title Configuration %}
```

File: source/_components/vallox.markdown [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+## {% linkable_title Fan Services %}
+
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_home` %}
+
+Set the fan speed of the `Home` profile.
+
+| Service data attribute | Optional | Description                                     |
+|------------------------|----------|-------------------------------------------------|
+| `entity_id`            |       no | Entitiy id of the fan, e.g.`fan.vallox`.        |
```
Comment (frenck) [inline source/_components/vallox.markdown]:
```suggestion
| `entity_id`            |       no | Entity id of the fan, e.g., `fan.vallox`.        |
```

File: source/_components/vallox.markdown [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+| `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_away` %}
+
+Set the fan speed of the `Away` profile.
+
+| Service data attribute | Optional | Description                                     |
+|------------------------|----------|-------------------------------------------------|
+| `entity_id`            |       no | Entitiy id of the fan, e.g.`fan.vallox`.        |
```
Comment (frenck) [inline source/_components/vallox.markdown]:
```suggestion
| `entity_id`            |       no | Entity id of the fan, e.g., `fan.vallox`.        |
```

File: source/_components/vallox.markdown [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+| `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_boost` %}
+
+Set the fan speed of the `Boost` profile.
+
+| Service data attribute | Optional | Description                                     |
+|------------------------|----------|-------------------------------------------------|
+| `entity_id`            |       no | Entitiy id of the fan, e.g.`fan.vallox`.        |
```
Comment (frenck) [inline source/_components/vallox.markdown]:
```suggestion
| `entity_id`            |       no | Entity id of the fan, e.g., `fan.vallox`.        |
```

**Review by frenck — CHANGES_REQUESTED** [review]
@andre-richter I've reviewed as well, and found some additional comments. Could you please take a look? Awesome! 👍

**Comment by andre-richter** [comment]
Sure, thanks for the pointers!

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/vallox.markdown [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.96
+ha_iot_class: Local Polling
+---
+
+The `vallox` component lets you control any Vallox ventilation unit that is supported by the [vallox_websocket_api](https://github.com/yozik04/vallox_websocket_api) (follow the link for a list of supported units).
+
+The component's fan platform allows you to either turn on/off the complete unit via the toggle switch or select a ventilation profile from a dropdown menu and therefore indirectly the fan speed. The four standard Vallox profiles are provided:
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown]: Please update this paragraph after the change regarding fan control in the parent PR.

File: source/_components/vallox.markdown [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+host:
+  description: The IP or hostname of the ventilation unit in your home network.
+  required: true
+  type: string
+name:
+  description: The name of this device as you want to see it in Home Assistant.
+  required: false
+  default: Vallox
+  type: string
+sensors:
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown]: This option was removed.

File: source/_components/vallox.markdown [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+  description: Include sensor reports. Optionally, provide a list of specific sensors.
+  required: false
+  default: All sensors.
+  type: list
+{% endconfiguration %}
+
+## {% linkable_title Fan Services %}
+
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_home` %}
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown]: Update service domain.

File: source/_components/vallox.markdown [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+
+Set the fan speed of the `Home` profile.
+
+| Service data attribute | Optional | Description                                     |
+|------------------------|----------|-------------------------------------------------|
+| `entity_id`            |       no | Entity id of the fan, e.g., `fan.vallox`.        |
+| `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_away` %}
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown]: As above.

File: source/_components/vallox.markdown [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+
+Set the fan speed of the `Away` profile.
+
+| Service data attribute | Optional | Description                                     |
+|------------------------|----------|-------------------------------------------------|
+| `entity_id`            |       no | Entity id of the fan, e.g., `fan.vallox`.        |
+| `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+### {% linkable_title Service `fan.vallox_set_profile_fan_speed_boost` %}
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown]: As above.

File: source/_components/vallox.markdown [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+
+- `fan_speed`
+- `temp_extract_air`
+- `temp_exhaust_air`
+- `temp_outdoor_air`
+- `temp_supply_air`
+- `humidity`
+- `remaining_time_filter`
+
+To activate _all_ sensors, add `sensors:` to your configuration:
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown]: Remove this section and below config examples.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_components/vallox.markdown [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.96
+ha_iot_class: Local Polling
+---
```
Comment (MartinHjelmare) [inline source/_components/vallox.markdown:15]: Is there no logo?

Comment (andre-richter) [reply source/_components/vallox.markdown:15]: What are legal implications of adding logos? I have no affiliation to the company whatsoever.

Comment (MartinHjelmare) [reply source/_components/vallox.markdown:15]: We have to check what the company policy on using logos is. Bigger companies usually have instructions on logo use.

Comment (andre-richter) [reply source/_components/vallox.markdown:15]: I'll have to pass on this one, already overshot my budget for working on this integration by too much, sorry :(

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/vallox.markdown (hunk 17 above)
Comment (klaasnicolaas) [reply source/_components/vallox.markdown:15]: I have not been able to find any policy, I think you can just add their logo.

Comment (frenck) [reply source/_components/vallox.markdown:15]: Will add the logo later.

File: source/_components/vallox.markdown (hunk 11 above)
Comment (frenck) [reply source/_components/vallox.markdown]: Addressed

**Comment by frenck** [comment]
Thanks for the PR @andre-richter! Well done 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**